editors/code/src/main.ts

```diff
@@ -33,7 +33,7 @@ export function outputChannel() {
 }
 
 export interface RustAnalyzerExtensionApi {
-    client: lc.LanguageClient;
+    client?: lc.LanguageClient;
```

Comment:
This is public API but I am not sure how we're supposed to version it.

Reply:
Well our versioning story is a bit awkward anyways in that 0.4 is nightly and 0.3 is stable. So minor version bumps don't really make sense for API stability either

Reply:
Since node uses standard semver anyways, I think every version we publish can include breaking changes, so we should be fine?